.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  "extends": ["airbnb", "prettier", "prettier/react"],
+  "plugins": ["react", "prettier"],
+  "parser": "babel-eslint",
+  "env": {
+    "browser": true,
+    "es6": true,
+    "node": true,
+    "jest": true,
```

Comment:
I think it's fine to have Jest in your `env` here, but if you want to you could put it in a separate `overrides` section of your eslintrc and set it up to only apply to test files.  That would mean that you'd still get errors if you try to call Jest functions in non-test files (where they won't work).  On the other hand, that isn't likely to come up that often, so might not be worthwhile